ndk-sys/generate_bindings.sh

```diff
@@ -1,10 +1,21 @@
 #!/bin/sh
 
-sysroot="${ANDROID_NDK_ROOT}"/toolchains/llvm/prebuilt/linux-x86_64/sysroot/
+os=$(uname -s)
+
+if [[ "$os" == "Darwin" ]]; then
+    host_tag="darwin-x86_64"
+elif [[ "$os" == "CYGWIN"* ]]; then
+    host_tag="windows-x86_64"
+else
+    host_tag="linux-x86_64"
+fi
+
+sysroot="${ANDROID_NDK_ROOT}"/toolchains/llvm/prebuilt/${host_tag}/sysroot/
 [ ! -d "$sysroot" ] && echo "Android sysroot $sysroot does not exist!" && exit 1
 
 while read ARCH && read TARGET ; do
     bindgen wrapper.h -o src/ffi_$ARCH.rs \
+        --no-size_t-is-usize \
```

Comment:
I think I _intentionally_ got rid of that in #370, any reason to keep it?

Reply:
I found `event.rs` is using `size_t`, so I thought `size_t` is needed for some reason, and I didn't know the existence of #370. The last version of `ffi_*.rs` still had `size_t` before I modified it without `--no-size_t-is-usize`, so I encountered compile errors due to lack of `size_t`. So I thought it is something like a platform-specific behavior of `bindgen`, because I'm using mac now.

I also think removing `size_t` is better, but without `--no-size_t-is-usize`, the current code still makes compile errors due to other modules.

Reply:
The change is now merged in as I think the next release will be breaking anyway (lots of things queued up), so you should be safe to rebase on `main` and drop most of the regen changes here except adding the new API.

Reply:
Got it. Font bindings are not depending on `size_t`, so `font.rs` will be OK even without `--no-size_t-is-usize`.

Reply:
I merged (I'm sorry, but I have a lot of bad memories of `rebase`, it always broke the whole branch) the newly updated `master` branch and removed `--no-size_t-is-usize` from `generate_bindings.sh`.